jetty-runtime/jetty9-base/pom.xml

```diff
@@ -122,7 +135,7 @@
           <arguments>
             <argument>-jar</argument>
             <argument>../jetty-distribution-${jetty.version}/start.jar</argument>
-            <argument>--add-to-startd=gae,http,jsp,jstl</argument>
+            <argument>--add-to-startd=http,deploy,jsp,jstl,http-forwarded,resources,gae</argument>
```

Comment:
Thanks Joakim! It looks like only the `http-forwarded` is relevant to the issue that this PR is resolving. I wonder if the other changes should  be moved to a separate PR.